include/wlr/types/wlr_output.h

```diff
@@ -277,4 +277,14 @@ enum wl_output_transform wlr_output_transform_invert(
 enum wl_output_transform wlr_output_transform_compose(
 	enum wl_output_transform tr_a, enum wl_output_transform tr_b);
 
+/**
+ * Marks the output as in need of a buffer swap.
+ */
+void wlr_output_update_needs_swap(struct wlr_output *output);
```

Comment:
This shouldn't be in this header. There's already a definition of this in `include/wlr/interfaces/wlr_output.h`. This function is intended to be used by backends, not compositors.